soc/arm/st_stm32/stm32h7/linker.ld

```diff
@@ -6,4 +6,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define RAM3_SIZE (64 * 1K)
```

Comment:
This won't be needed, imho, if we use device tree for getting the RAM3 address and size.

Reply:
Done.